package/libyui-gtk.spec

```diff
@@ -84,8 +84,8 @@ This package has very few dependencies.
 
 %build
 
-export CFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations"
-export CXXFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations"
+export CFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations -fPIC"
+export CXXFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations -fPIC"
```

Comment:
No details, it simply failed at the link phase with an error saying something like `recompile with -fPIC option`. After adding it passed so I did not go into the details... :wink: